core/server/master/src/main/java/alluxio/master/file/meta/InodeTreeBufferedIterator.java

```diff
@@ -84,12 +84,9 @@
   public static CloseableIterator<Journal.JournalEntry> create(InodeStore inodeStore,
       InodeDirectory rootInode) {
     InodeTreeBufferedIterator iterator = new InodeTreeBufferedIterator(inodeStore, rootInode);
-    return new CloseableIterator<Journal.JournalEntry>(iterator) {
-      @Override
-      public void closeResource() {
-        iterator.close();
-      }
-    };
```

Comment:
In the updated interface, the caller is required to pass a close operation instead of implementing the method. I personally think this one will be cleaner.